### DISCUSSION TITLE
docker container

### DISCUSSION THREAD

**Discussion by kukiiii** [sidorares/node-mysql2#1468] [Q&A]
Can I install this node-mysql2 on docker container ?  And if I can ,can you explain me how to do it, I'm asking for my hosting cuz the guy who is the owner of the hosting, told me he doesn't know how to do it.

**Comment by sidorares** [comment]
yes, there is nothing special about node-mysql2 in terms of peer dependencies - there is only pure JS deps in the tree so nothing extra to configure for your container - just follow same steps as per any other "how to run node app with docker" tutorial

Can you be a bit more specific about what are you trying to achieve and what issues you have? "Can I install this node-mysql2 on docker container" is a bit strange question, `npm install` script in dockerfile will add node-mysql2 to your container node_modules during the build

**Comment by kukiiii** [comment]
Would you like to give me some instructions on how to install it, i followed the npm install --save mysql2 but i have no idea on how to proceed. Could you help me?

**Comment by sidorares** [comment]
Assuming you already have installed docker on your dev machine. If not - follow instructions at https://docs.docker.com/get-docker/

If you want to start mysql __server__ with docker: https://hub.docker.com/_/mysq

To develop node code running in the container that is connecting to mysql follow steps here - https://nodejs.org/en/docs/guides/nodejs-docker-webapp/
